CHANGES.md

```diff
@@ -1,6 +1,41 @@
 # grmtools 0.11.0 (XXXX-YY-ZZ)
 
-## APIs deprecated
+## API deprecation summary
+
+* Several of the functions / structs surrounding the compile-time construction
+  of grammars have changed: more details are given below, but in most cases
+  a `build.rs` that looks as follows:
```

Comment:
~~that~~

Reply:
I think "that" is correct here, because we're making clear that we're only talking about `build.rs` files which follow this pattern.

Reply:
The sentence still sounds weird: more details are given below, but in most cases a `build.rs` that looks as follows.

What's the "that" for?

Reply:
There's a follow-on sentence after the code block so it's effectively "in most cases
  a `build.rs` that looks as follows <...> can be changed to <...>".

Reply:
Ooooohhhhh! Ignore me. :-)